pkg/provision/rolling.go

```diff
@@ -84,15 +86,17 @@ func RollingUpdate(platform *platform.Platform, minAge time.Duration, drainTimeo
 
 		log.Infof("Replaced %s in %s", node.Name, timer)
 
-		doUntil(func() bool {
+		if succeededWithinTimeout := doUntil(timeout, func() bool {
```

Comment:
linter was unhappy with the `false` result (for when the check failed until timeout) was never used.